kong.conf.default

```diff
@@ -1328,6 +1328,7 @@
                                  # - `strict`: the router will be rebuilt
                                  #   synchronously, causing incoming requests to
                                  #   be delayed until the rebuild is finished.
+                                 #   (This is deprecated and will be removed soon)
```

Comment:
IMO `soon` sounds a little vague. What do you think about `[...] will be removed in a future release` or something similar?

Reply:
done